src/layouts/authors.ts

```diff
@@ -0,0 +1,18 @@
+export const authors = [
+  {
+    name: "Faraz Patankar",
+    image: "https://og.railway.app/authors/faraz-patankar.jpeg",
+  },
+  {
+    name: "Jake Cooper",
+    image: "https://og.railway.app/authors/jake-cooper.jpeg",
```

Comment:
Should these images live somewhere else?

Reply:
I think it is fine how where it is. Alternatively, we could make the author image and name a text input. But we can add that if we need it in the future.